python/django_rusty_templates/__init__.py

```diff
@@ -4,6 +4,8 @@
 
 from .django_rusty_templates import Engine, Template
 
+__all__ = [Engine, Template]
```

Comment:
These should all be strings. I also think we want to export `RustyTemplates`, but not `Engine`.

```suggestion
__all__ = ["RustyTemplates", "Template"]
```

Reply:
Damn, you're right. Goes to show that I very rarely use `__all__`.